src/renderer/features/flexible-change-signatories/model/change-signatories-model.ts

```diff
@@ -179,7 +181,29 @@ const { $tx, $fee, $pendingFee } = createComplexTxStore({
   transaction: $coreTx,
 });
 
-const validator = createTxValidator();
+//todo move to ... probably should have a file "validators" in the feature
+const validator = createTxValidator({
+  additionalBalanceRules: [
+    ({ route, getBalance, asset, api }) => {
+      const initiator = accountService.findInitiator(route);
+      if (!initiator || !accountUtils.isFlexibleMultisigAccount(initiator)) return;
+
+      const balance = getBalance(initiator.accountId, initiator.chainId, asset.assetId);
+
+      if (nullable(balance)) return;
+
+      const deposit = proxyService.getProxyDeposit(api, '0', 1);
```

Comment:
This is not correct - at this point, there is already a proxy deposit made, so the runtime wont charge use the full `deposit + factor` bur rather just `factor`

This can be accounted by passing already existing deposit instead of `'0'` here
Also we should use `proxyNumber + 1` here since we cannot guarantee this flex msig has just 1 proxy, not to mentioned it will be 1, not 2, since we are adding second proxy in the simplest case